src/app/controllers/admin-console.server.controller.js

```diff
@@ -20,7 +20,7 @@ const Form = getFormModel(mongoose)
 const _ = require('lodash')
 
 const logger = require('../../config/logger').createLoggerWithLabel(module)
-const { getRequestIp, getTrace } = require('../utils/request')
```

Comment:
`getTrace` is unused anymore right? Should remove all the unused imports in the other files too

Reply:
edited accordingly